slam-libraries/viam-cartographer/CMakeLists.txt

```diff
@@ -51,6 +51,8 @@ endif()
 
 file(GLOB_RECURSE ALL_VIAM_HDRS "src/*.h")
 file(GLOB_RECURSE ALL_VIAM_SRCS "src/*.cc")
+file(GLOB_RECURSE ALL_VIAM_TEST_SRCS "src/*_test.cc")
+list(REMOVE_ITEM ALL_VIAM_SRCS ${ALL_VIAM_TEST_SRCS})
```

Comment:
As long as test files are named with `*_test.cc` and are located within the `src` folder, they'll be included when running `./scrips/test_cartographer.sh`.